src/main/frontend/worker/db/migrate.cljs

```diff
@@ -611,6 +611,43 @@
    {:db/ident :logseq.property.view/type.gallery
     :logseq.property/icon {:type :tabler-icon :id "layout-grid"}}])
 
+(defn- migrate-views
+  [conn _search-db]
+  (let [db @conn
+        tags (->> (d/datoms db :avet :block/tags :logseq.class/Tag)
+                  (map (fn [d] (d/entity db (:e d)))))
+        properties (->> (d/datoms db :avet :block/tags :logseq.class/Property)
+                        (map (fn [d] (d/entity db (:e d)))))
+        tx-data (mapcat
+                 (fn [e]
+                   (let [id (:db/id e)
+                         ks [:logseq.property.table/sorting
+                             :logseq.property.table/filters
+                             :logseq.property.view/group-by-property
+                             :logseq.property.table/hidden-columns
+                             :logseq.property.table/ordered-columns
+                             :logseq.property.table/sized-columns
+                             :logseq.property.table/pinned-columns
+                             :logseq.property.view/type]
+                         view-properties (->> (keep (fn [k] (when-let [v (get (d/pull db [k] id) k)]
+                                                              [k v])) ks)
+                                              (into {}))]
+                     (when (seq view-properties)
+                       (let [view-page-id (:db/id (ldb/get-page db common-config/views-page-name))
+                             _ (when (nil? view-page-id)
+                                 (throw (ex-info "View page not exists" {})))
+                             view (-> (assoc view-properties :logseq.property/view-for id)
+                                      (merge {:block/uuid (common-uuid/gen-uuid)
```

Comment:
If different clients execute this function(under rtc env),
it will generate similar blocks with different block/uuid. Is this acceptable?

Reply:
Handled by https://github.com/logseq/logseq/pull/11739/commits/a7f81ffe47a86ccd85e21efdcf0a8b41f4709175.